com.unity.netcode.gameobjects/CHANGELOG.md

```diff
@@ -6,6 +6,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 
 Additional documentation and release notes are available at [Multiplayer Documentation](https://docs-multiplayer.unity3d.com).
 
+== [Unreleased]
```

Comment:
Shouldn't all of these additions should match the same format as 1.0.0-pre.5 and below?

Reply:
... TIL the font I'm using makes # look like =.